tests/test_unifi_client.py

```
@@ -0,0 +1,343 @@
+    assert len(devices) == 1
+    assert isinstance(devices[0], Device)
+    assert devices[0].mac == "00:11:22:33:44:55"
+
+    # Test get_device()
+    device = unifi_client.get_device("00:11:22:33:44:55")
+    assert isinstance(device, Device)
+    assert device.mac == "00:11:22:33:44:55"
+
+
+@patch("requests.Session.request")
+def test_client_management(mock_request, unifi_client, mock_client_response):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_client_management`

Add return type annotation: `None`

(ANN201)